vfs/disk_usage_netbsd.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The LevelDB-Go and Pebble Authors. All rights reserved. Use
```

Comment:
Let's update the copyright to this year, 2023.

Reply:
Sure!